File: script/match/genosum.m
%%% genosum.m
%%% Calculate the minor allele counts for a chromosome
%
data=load('matched.Geno_chrXXX.data');
ma=sum(data,1);   %? ma=sum(data,2);
dlmwrite('matched.Geno.ma_chrXXX',ma,' ');

File: script/adj/genosum.R
## genosum.R
##Calculate the minor allel counts for a chromosome
#
library(data.table)
data <- fread("matched.Geno_chrXXX.data");
ma <- as.matrix(t(colSums(data)))
fwrite(ma, "matched.Geno.ma_chrXXX", sep=" ", quote=F, col.names=F, row.names=F);
